website/content/en/docs/config/network.md

```diff
@@ -148,8 +148,8 @@ writable by `root` and will reject `socket_vmnet` installed by Homebrew into a u
 # This assumes that Xcode Command Line Tools are already installed
 git clone https://github.com/lima-vm/socket_vmnet
 cd socket_vmnet
-# Change "v1.1.5" to the actual latest release in https://github.com/lima-vm/socket_vmnet/releases
-git checkout v1.1.5
+# Change "v1.2.0" to the actual latest release in https://github.com/lima-vm/socket_vmnet/releases
+git checkout v1.2.0
```

Comment:
Change looks fine, but since we don't build latest version from git, why not use the latest release tarball?

Reply:
I don't get. The binary releases are built from the git repo.

Reply:
Sure the result is the same, but we don't depend on being able to build socket_vment. We use the same method used by users to install it, which is much simpler. Get the latest release from github, download the tarball, and extract it. And with this change we can remove this configuration and avoid this another detail we have to maintain forever.

Reply:
I don't want to block this simple change, just a thought on the next step.

Reply:
> I don't want to block this simple change, just a thought on the next step.

Ok, I've approved and merged then.

Please file a new issue/PR for using the upstream release instead. I do see the (minor) point that this validates that the releases can be installed and used. Also means that we don't need knowledge about how to build `socket_vmnet` in the `lima` repo. But then that is pretty much abstracted away in the `Makefile` anyways.